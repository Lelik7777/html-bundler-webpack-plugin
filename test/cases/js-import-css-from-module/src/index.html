<!DOCTYPE html>
<html>
<head>
  <title>Home</title>
  <script src="./main.js" defer="defer"></script>
</head>
<body>
  <h1>Home</h1>
  <div class="apple">apple</div>
  <div class="test-fixture-primary">test-fixture-primary</div>
  <div class="img image-bg">image</div>
</body>
</html>