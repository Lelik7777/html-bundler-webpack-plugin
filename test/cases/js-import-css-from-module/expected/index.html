<!DOCTYPE html>
<html>
<head>
  <title>Home</title>
  <link href="assets/css/main.935cb8bd.css" rel="stylesheet">
<script src="assets/js/main.b8d3babd.js" defer="defer"></script>
</head>
<body>
  <h1>Home</h1>
  <div class="apple">apple</div>
  <div class="test-fixture-primary">test-fixture-primary</div>
  <div class="img image-bg">image</div>
</body>
</html>